crates/autopilot/src/infra/orderbook/auction.rs

```diff
@@ -0,0 +1,18 @@
+use crate::{
+    domain,
+    infra::{self, persistence::auction::dto},
+};
+
+impl infra::Orderbook {
```

Comment:
Let's merge with parent impl for a single view on all it's parts.

Reply:
merged